libsql/src/hrana/mod.rs

```diff
@@ -38,9 +38,12 @@ struct Cookie {
 }
 
 pub trait HttpSend: Clone {
-    type Stream: Stream<Item = Result<Bytes>> + Unpin;
+    type Stream: Stream<Item = std::io::Result<Bytes>> + Unpin;
     type Result: Future<Output = Result<Self::Stream>>;
     fn http_send(&self, url: Arc<str>, auth: Arc<str>, body: String) -> Self::Result;
+
+    /// Schedule sending a HTTP post request without waiting for the completion.
+    fn oneshot(self, url: Arc<str>, auth: Arc<str>, body: String);
```

Comment:
As usual: cloudflare doesn't operate on tokio and has its own way of spawning tasks. Since this is called from `Drop::drop` it cannot be specialised between CloudflareSender and regular Hyper sender.